irohad/ametsuchi/command_executor.hpp

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_AMETSUCHI_COMMAND_EXECUTOR_HPP
+#define IROHA_AMETSUCHI_COMMAND_EXECUTOR_HPP
+
+#include <boost/format.hpp>
+
+#include "common/result.hpp"
+#include "interfaces/common_objects/types.hpp"
+
+namespace shared_model {
```

Comment:
I think in the future it would be beneficial to move such declarations to separate header inside shared_model (not related to this pr, just a thought)